src/App.tsx

```diff
@@ -80,7 +80,7 @@ export default function () {
                   <Thread />
                 </Route>
                 <Route path="/404">
-                  <NotFound />
+                  <LazyComponent lazyImported={<NotFound />} />
```

Comment:
Why do you need `Lazy import` here? There is no need in it

Reply:
Probably we don't need it